### PR TITLE
Redesigned Meditor

### LINKED ISSUE #706: Implement redesigned meditor
Issue to track implementation of the redesigned meditor.

### LINKED ISSUE #789: Meditor must not freeze web ui
It might be just too long, but must be beyond 30 sec as we do get through on 000026 with slightly over 30 sec (see https://github.com/dandi/dandi-api-webshots/#000026)  but browser (when trying locally directly without webshots) does just say that Aw Snap eventually)

### PR DESCRIPTION
**Pull request by mvandenburgh**
Implement the redesigned meditor. This also has the side effect of fixing all the slow loading/freezing problems we've had.

Closes #706
Closes #789

### CODE REVIEW

**Comment by mvandenburgh** [comment]
This is not done yet, I opening this just to see how this works with staging data - I still have to clean up some styling and fix some things.

**Review by AlmightyYakob — CHANGES_REQUESTED** [review]
A couple of suggestions, but otherwise looks good! Great job with this

File: web/src/components/Meditor.vue [hunk 1]
```diff
@@ -119,100 +100,93 @@
...
+              class="px-7"
             >
-              <v-jsf
-                ref="complexRef"
-                :value="complexModel[propKey]"
-                :schema="complexSchema.properties[propKey]"
+              <v-jsf-wrapper
+                :transaction-tracker="transactionTracker"
+                :prop-key="propKey"
+                :editor-interface="editorInterface"
```
[the remark this answers is not in the document]
Comment (mvandenburgh) [reply web/src/components/Meditor.vue:172]: Sounds good to me, I'll implement this.

File: web/src/components/VJsfWrapper.vue [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+          <v-icon>mdi-arrow-right</v-icon>
+        </v-btn>
+        <v-btn
+          v-else
+          class="grey darken-3 white--text"
+          elevation="0"
+          :disabled="!formValid"
+          @click="saveItem(propKey)"
+        >
+          <span class="mr-1">Save Item</span>
```
[the remark this answers is not in the document]
Comment (mvandenburgh) [reply web/src/components/VJsfWrapper.vue:142]: Done https://github.com/dandi/dandi-archive/pull/862/commits/f3c38a00b4aff20b6628bc45857624df86aa4b5d

File: web/src/components/Meditor.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (mvandenburgh) [reply web/src/components/Meditor.vue:172]: Punting this to a future PR as we discussed offline. Issue https://github.com/dandi/dandi-archive/issues/894

**Review by AlmightyYakob — APPROVED** [review]

**Pull request merged**